microsoft_authentication/auth/auth_utils.py

```diff
@@ -88,11 +88,14 @@ def validate_email(email):
     return "@" in email and email.split("@")[1] in settings.MICROSOFT["valid_email_domains"]
 
 
-def get_django_user(email, create_new=True, is_staff=True):
+def get_django_user(email, first_name, last_name, create_new=True, is_staff=True):
```

Comment:
```suggestion
def get_django_user(email, first_name=None, last_name=None, create_new=True, is_staff=True):
```

We only need to update if the value is not None